demo/app/README.md

```diff
@@ -29,14 +25,25 @@ Things you need to install before running the app in your local:
       [✓] HTTP Host Availability
     ```
 - The Dart SDK is bundled with Flutter; it is not necessary to install Dart separately.
-- For ios setup - install [XCode](https://docs.flutter.dev/get-started/install/macos#install-xcode)
-- Trustbloc app uses the plugin, therefore we uses third-party CocoaPods dependency manager `sudo gem install cocoapods`[See More here][https://docs.flutter.dev/get-started/install/macos#deploy-to-ios-devices]
-  - Once we clone the project, run `pod install` to get all the latest dependencies.
-- For android setup - install [Android Studio](https://docs.flutter.dev/get-started/install/macos#android-setup)
-- Build the sdk bindings by following this [document](https://github.com/trustbloc/wallet-sdk/blob/main/cmd/wallet-sdk-gomobile/README.md)
-- Either run the emulator for android or attach the usb device, android studio will detect the device to run the app on your phone/test device.
-- Frequent Issue and solutions:  
+- iOS
+  - For ios setup - install [XCode](https://docs.flutter.dev/get-started/install/macos#install-xcode)
+  - TrustBloc app uses the plugin, therefore we uses third-party CocoaPods dependency manager `sudo gem install cocoapods`[See More here][https://docs.flutter.dev/get-started/install/macos#deploy-to-ios-devices]
+  - Apple Developer Account
+- Android
+  - For android setup - install [Android Studio](https://docs.flutter.dev/get-started/install/macos#android-setup)
 
+### Build and Run
+- Build the SDK Bindings by following this [document](https://github.com/trustbloc/wallet-sdk/blob/main/cmd/wallet-sdk-gomobile/README.md)
+- Android
+  - Open the app in Android Studio
+  - Either run the Emulator for Android or attach the usb device, Android studio will detect the device to run the app on your phone/test device.
+- iOS
+  - run `pod install`
+  - Open the app in XCode
+  - Install app to Simulator or Device from the XCode Build tab
```

Comment:
you need to run `flutter run` or `flutter build` before you will be able to run the app directly from XCode.

Reply:
[Updated](https://github.com/trustbloc/wallet-sdk/compare/4eae35aba5733d626e32784e32b2f50a927d6728..d5dd7f8528cdaba00b420d021d2ff873e144794c)